client/components/ReviewRating/RatingBreakDown/RatingBreakDown.jsx

```diff
@@ -86,15 +86,17 @@ const RatingBreakDown = ({
               <span>
                 {starFilter.map((val) => (
                   <div key={val}>
-                    {String(val)}
-                    {' '}
-                    star
+                    <p>
+                      {String(val)}
```

Comment:
Seems like you could use a template literal here `${String(val)} star` rather that breaking this up on to different lines and having empty quotes.